### PR TITLE
Create Mistral test

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description
This will allow adding any model
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)  
- [x] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update  
- [ ] Chore: cleanup/renaming, etc  
- [ ] RFC  
## Acceptance

- [ ] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [ ] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: app/api/huggingface/mistral/route.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      "https://api-inference.huggingface.co/models/mistralai/Mistral-7B-v0.1",
+      {
+        headers: { Authorization: `Bearer ${process.env.HUGGINGFACE_TOKEN}` },
+        method: "POST",
+        body: JSON.stringify(req.prompt),
+      }
+    );
+    const result = await response.json();
+    console.log(result);
+
```
[the remark this answers is not in the document]
Comment (EstebanDalelR) [reply app/api/huggingface/mistral/route.ts:28]: Expected, closing.

[the remark this answers is not in the document]
Comment (baristaGeek) [reply app/api/huggingface/mistral/route.ts:28]: Sure, I trust you

**Comment by EstebanDalelR** [comment]
> Would this be the route to test?
> 
> http://localhost:3000/api/huggingface/mistral/route.ts
> 
> I'm testing with that route and
> 
> ```
> {
>     "email": "estebanvargas94@gmail.com",
>     "prompt": "Build something with this random prompt"
> }
> ```

No, the route is `http://localhost:3000/api/huggingface/mistral`, as per the [NextJS docs](https://nextjs.org/docs/app/building-your-application/routing/route-handlers)

**Comment by EstebanDalelR** [comment]
Closing due to cost

**Pull request closed without merging**